dace/sdfg/replace.py

```diff
@@ -148,6 +151,14 @@ def replace_properties_dict(node: Any,
                     propval[symname] = symbolic.pystr_to_symbolic(str(sym_mapping)).subs(symrepl)
                 except AttributeError:  # If the symbolified value has no subs
                     pass
+        elif isinstance(propclass, properties.ListProperty) and pname == 'params':
```

Comment:
@tbennun This seems to be a bug: When renaming a parameter of a map, the parameters gets renamed correctly in the scope of the map (memlets, etc) but doesn't get renamed in the map definition.

If I add these lines, many of our tests start to break